packages/amplify_core/lib/src/types/datastore/conflict_handler/datastore_conflict_handler.dart

```diff
@@ -49,7 +49,7 @@ class ConflictResolutionDecision {
         "}";
   }
 
-  Map<String, dynamic> toJson() => {
+  Map<String, dynamic> toJson() => <String, dynamic>{
```

Comment:
Pro-tip: use `Map<String, Object?>` as a replacement for any `<String, dynamic>` code you find. It's the same thing but will prevent pesky dynamic errors.